playbook.yml

```diff
@@ -3,6 +3,8 @@
 - name: Install thirdparties
   hosts: trento-server
   become: true
+  vars:
+    install_method: "rpm"
```

Comment:
I'm not convinced about having it here, this way it has added visibility but I'm forced to repeat it in multiple places. Would this better go inside the roles or as group or hosts vars?

Reply:
Why don't we move the docker dependencies installation to the `containers` role?
Do we use docker somewhere else?
I mean, i think it makes sense